react/features/chat/middleware.js

```diff
@@ -226,26 +228,42 @@ function _addChatMsgListener(conference, store) {
     conference.on(
         JitsiConferenceEvents.MESSAGE_RECEIVED,
         (id, message, timestamp) => {
+            const isGif = isGifMessage(message);
```

Comment:
since we have code duplication on `JitsiConferenceEvents.MESSAGE_RECEIVED` and `JitsiConferenceEvents.PRIVATE_MESSAGE_RECEIVED`, with the privateMessage: true/false being the only diff, we could extract the handler in a `function _onConferenceMessageReceived(id, message, timestamp, private){...}`